docs/notebooks/eau_precpitable_CanESM5/.zattrs

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
There are quite a few files in `docs/notebooks` that should not be there. An error during testing?

Reply:
It could be that they weren't adequately listed in the `.gitignore`?

Reply:
They seem to be PMP-related. I think that it's just a mistake.

Reply:
I'm checking where those files are coming from